aboutus.html

```diff
@@ -84,7 +85,7 @@ <h1>Sierra Laney<br>President/Chair</h1>
                         <h2>Computer Science Major</h2>
                     </div>
                 </div>
-                <a class="button button-primary" href="http://livartly.com">PRESIDENT</a>
+                <a class="button button-primary" href="404.html">PRESIDENT</a>
```

Comment:
Couldn't you just removed the href?

Reply:
I realize that might cause some styling issues, but I think a 404 redirect here sends the wrong message to the user. If I get a 404, I think, "Oh, that page is somewhere but the link is broken, or it's been deleted, or something". This is a bit of a different case. Here its more like "That object shouldn't be clickable".

It's my opinion that you should never intentionally give the user a 404. A 404 just gives you a safe path out of any potential edge cases or unexpected user behaviour.